distributed/core.py

```diff
@@ -132,12 +132,12 @@ class Server(TCPServer):
     default_port = 0
 
     def __init__(self, handlers, max_buffer_size=MAX_BUFFER_SIZE,
-            connection_limit=512, **kwargs):
+            connection_limit=512, ssl_context_creator= create_ssl_context, **kwargs):
         self.handlers = assoc(handlers, 'identity', self.identity)
         self.id = str(uuid.uuid1())
         self._port = None
         self.rpc = ConnectionPool(limit=connection_limit)
-        super(Server, self).__init__(max_buffer_size=max_buffer_size, **kwargs)
+        super(Server, self).__init__(max_buffer_size=max_buffer_size, ssl_options=ssl_context_creator(), **kwargs)
```

Comment:
ahh. i see. I dont have to configure it after some `Server` state. The reason it is a method was mostly because it lets me test the server object with a trivial method.

would this be better:
```python
def __init__( ...):


    ssl_options = ssl_context_creator() 
    ...
    super(Server, self).__init__(..., ssl_options=ssl_options)
```